### PR TITLE
logos-on-username

### PR DESCRIPTION
**Pull request by pathak-arpita**

### CODE REVIEW

**Comment by alphainfinitus** [comment]
image
icons are too big, please match the size to design

File: src/ui-components/IdentityBadge.tsx [hunk 1]
```diff
@@ -52,16 +56,16 @@ const IdentityBadge = ({ className, address, identity, flags, web3Name }: Props)
...
 
 	const popupContent = <StyledPopup>
-		{identity?.legal && <li><span className='desc'>legal:</span>{identity.legal}</li>}
-		{identity?.email && <li><span className='desc'>email:</span>{identity.email}</li>}
-		{(identity?.judgements?.length || 0) > 0 && <li><span className='desc'>judgements:</span><span className='judgments'>{displayJudgements}</span></li>}
-		{identity?.pgp && <li><span className='desc'>pgp:</span>{identity.pgp}</li>}
-		{identity?.riot && <li><span className='desc'>riot:</span>{identity.riot}</li>}
-		{identity?.twitter && <li><span className='desc'>twitter:</span>{identity.twitter}</li>}
-		{identity?.web && <li><span className='desc'>web:</span>{identity.web}</li>}
+		{identity?.legal && <li><span className='desc'>legal:</span><span className='text-[12px]'>{identity.legal}</span></li>}
```
Comment (alphainfinitus) [inline src/ui-components/IdentityBadge.tsx:59]: arbitrary values being used, please used standardised font sizes

**Review by alphainfinitus — CHANGES_REQUESTED** [review]
Please check comments

File: src/ui-components/IdentityBadge.tsx (hunk 1 above)
Comment (sahas-01) [reply src/ui-components/IdentityBadge.tsx]: Resolved, pls check

**Comment by sahas-01** [comment]
> Please check comments

Resolved, pls check

**Comment by rajdeep7Singh** [comment]
Reverting this PR due to issues on Prod. Please check @This-Is-Prince

**Pull request merged**